### DISCUSSION TITLE
DAG looks planned in logging but not in calendar or actually being run.

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#33312] [Q&A]
We schedule DAG's with the CronTriggerTimetable schedule option to run on certain days on certain times.
99% of the times this goes good. But in some cases the DAG is not scheduled.

In the logging i see a scheduling message but in the calendar overview its not there and its not executed.
Log entries show a new sechedule after a (seems to be succesfull) run on the 28th
![image](https://github.com/apache/airflow/assets/19835721/9f4ca7bd-c321-4471-8973-a69520beec66)

Calender shows no status or planned object
![image](https://github.com/apache/airflow/assets/19835721/fdb40a9a-5b70-4597-92b4-037d762dab71)

Has this anything to do with https://github.com/apache/airflow/issues/31687

**Comment by potiuk** [comment]
Quite likely it's the same issue as #31687
